libvast/builtins/connectors/directory.cpp

```diff
@@ -51,7 +51,19 @@ class plugin : public virtual saver_plugin {
                       info.input_schema.make_fingerprint(), info.format);
     auto new_args = std::vector<std::string>{file_path.string()};
     // TODO: handle overwrite semantics
-    return saver_plugin_->make_saver(new_args, std::move(info), ctrl);
+    auto file_saver
+      = saver_plugin_->make_saver(new_args, std::move(info), ctrl);
+    auto guard = caf::detail::make_scope_guard([=] {
+      fmt::print(stdout, "{}\n", file_path.string());
```

Comment:
This unconditional printing means that we also print the path if the pipeline is aborted. But perhaps that's fine.

Reply:
We still create the file in that case, right?

Reply:
I added a note to the code and left the behavior as-is.